files/pt-br/web/css/float/index.md

```diff
@@ -92,15 +92,15 @@ h4 {
 </div>
 ```
 
-{{ EmbedLiveSample('Exemplos', '', '', '', 'Web/CSS/float') }}
+{{EmbedLiveSample('','400','180')}}
```

Comment:
This change seems to be unrelated?

Reply:
The old line was giving me error in PR-test, so I changed like found in US version page.